SMLHelper/Patchers/CraftDataPatcher_BelowZero.cs

```diff
@@ -39,7 +50,7 @@ private static void AddCustomTechDataToOriginalDictionary()
                 bool techDataExists = TechData.entries.ContainsKey(techType);
                 if (techDataExists && TechData.entries[techType] != CustomTechData[techType])
                 {
-                    if (TechData.TryGetValue(techType, out JsonValue originalData))
+                    if (TechData.entries.ContainsKey(techType))
```

Comment:
`(TechData.entries.ContainsKey(techType)` is checked twice here.  
This nested if-block look redundant when `techDataExists` already has the value it is checking.

Reply:
yes it is. altered this a bit.